tools/kms_secrets_encryption.py

```diff
@@ -7,13 +7,48 @@
 
 import argparse
 import base64
+import importlib
 import os
-from google.cloud    import kms_v1
-from google.oauth2   import service_account
+import subprocess
 
 
 SECRETS_START_FLAG = "# <-- Start of secrets section, do not edit this line. -->"
 
+
+def import_or_install_module(pip_package, module_name = None):
+    """A function that imports a Python top-level package or module. 
+    If the required package is not installed, it will install the package before importing it again.
+
+    Args:
+        pip_package (str): the name of the pip package to be installed
+        module_name (str): the import name of the module if it is different than the pip package name
+
+    Returns:
+        module: the top-level package or module
+    """
+
+    if module_name is None:
+        module_name = pip_package
+
+    try:
+        module = importlib.import_module(module_name)
+        print(f"Successfully imported {module_name}.")
+
+    except ImportError:
+        install_cmd = f"pip3 install {pip_package} --user"
```

Comment:
Can we add a prompt for user confirmation? Something like: "this script requires <pip-package-name> but is not available.  Install package by running "pip3 install {pip-package-name} --user"? (Y/N)" and if user enters N then quit.